cpp/include/cugraph/experimental/graph_view.hpp

```diff
@@ -228,9 +232,9 @@ size_t constexpr num_sparse_segments_per_vertex_partition{3};
 
 // Common for both graph_view_t & graph_t and both single-GPU & multi-GPU versions
 template <typename vertex_t, typename edge_t, typename weight_t>
-class graph_base_t {
+class graph_base_t : public graph_envelope_t::base_graph_t /*<- visitor logic*/ {
```

Comment:
Not directly applicable to this line...

This (and much of the CD code) is in the experimental namespace.  It feels like this should go into a permanent namespace.  Again, not sure of a good name, `api` perhaps?  Probably should be consistent with whatever we name the directory.

Reply:
Similar answer to above. However, I am opened to the idea of placing all this logic in a separate namespace. But for reasons discussed above, I would not call it `api`. Maybe `namespace visitor` or `namespace dispatch` ?

Reply:
Perhaps based on your above argument `namespace visitor` would be appropriate here.

Reply:
Done.